contrib/osc.complete

```diff
@@ -58,6 +58,22 @@ let count--
 test "${COMMAND_LINE:$last}" = " " && let count++
 unset last
 
+XDG_CACHE_HOME="${XDG_CACHE_HOME:-$HOME/.cache}"
+XDG_CONFIG_HOME="${XDG_CONFIG_HOME:-$HOME/.config}"
+
+for xdg_dir in "${XDG_CACHE_HOME}" "${XDG_CONFIG_HOME}"; do
```

Comment:
You can skip line 61 and 62 by writing it like this: `"${XDG_CACHE_HOME:=$HOME/.cache}"`

Reply:
Possibly, but I prefer setting the defaults explicitly like this rather than doing it elsewhere with a side-effect of setting the default.